shell/probe/demo.ps1

```diff
@@ -131,7 +153,7 @@ Write-Host "
 ###########################################################################################################
 "
 
-if ($Status -eq 100 ) {
+if ($Status -in 100,127 ) {
```

Comment:
more codes then this exist... we'd be better off putting all codes in here and printing one of four conditions:
- passed
- failed
- error
- defense caught

Reply:
or at minimum; combining passed and defense code codes into "worked"